tests/models.py

```diff
@@ -36,6 +36,7 @@ class Article(models.Model):
         PeriodicalPublisher, related_name="published", on_delete=models.CASCADE
     )
     release = models.DateField()
+    additional_info = models.CharField(max_length=100, null=True, default=None)
```

Comment:
Adding an additional field seems to have broken some of the tests -- would it be simpler to make one of the current fields nullable and add a new item to the list? Maybe the `release` field?

Reply:
I will change `release` field to null then try again. Thanks for reviewing.

Reply:
Hi @clokep  I have a question. I found that I was going with the wrong order of Postgres, and then I changed it. But 
when I use the logic return `1` if `value1` is `None`, `-1` if `value2` is `None` and `0` if both `None`. The result is not returned as expected. The None value isn't sorted properly.  
It's return `[None, datetime.date(2001, 6, 12), None, datetime.date(1980, 4, 21), datetime.date(1990, 8, 14)]` when I sorted by `("release")`. Do you know any other place that possible has an issue?

Reply:
I can't think of anything off the top of my head. Is this with the latest code in this branch?

Reply:
No, the current one is the reverse version. The reverse version seems to work fine. It's just not the Postgres order.

Reply:
Can you push your latest code so I can take a look? Thanks!

Reply:
@clokep I updated my latest code. The 2 test cases that I define still failed because of the above issue. Please take a look. Thank you.

Reply:
Hi @clokep, Have you take a look at my code?

Reply:
I did -- I was able to reproduce the behavior of SQLite vs. Postgres and I'm not really sure what to do about it, I couldn't see an easy way in the code of:

1. Always returning them at the beginning or end (and just defining a behavior), due to the lazy nature of the iterators.
2. Checking if the underlying database sorts nulls first or last.

Without those I'm not really sure where to go.

It *might* be doable to hard-code a mapping of database -> expected ordering, but that sounds finicky.